firewood/src/storage/buffer.rs

```diff
@@ -569,7 +570,7 @@ impl DiskBufferRequester {
             .send(BufferCmd::GetPage((space_id, page_id), resp_tx))
             .map_err(StoreError::Send)
             .ok();
-        resp_rx.blocking_recv().unwrap()
+        block_in_place(move || resp_rx.blocking_recv().unwrap())
```

Comment:
Just guessing that you don't need this here because you're not in an async function

Reply:
+1 will it work if not in an async call?

Reply:
Without this, all the tests fail. This code does nothing if you're not in an async context. If you are, then it moves you to a blockable thread to prevent interference with the async runtime.